### PR TITLE
Show unwinding error type in callstack view

### PR DESCRIPTION
**Pull request by florian-kuebler**
This adds the unwinding error type to the the callstack
view's text. Further, it adds a tooltip describing the
error more detailed.

http://screenshot/C85Tio98X5hSQCZ

Test: Take capture, select broken callstack, show tooltip.
Bug: http://b/190590422

### CODE REVIEW

File: src/OrbitQt/orbitsamplingreport.ui [hunk 1]
```diff
@@ -103,7 +103,7 @@
                <string/>
               </property>
               <property name="icon">
-               <iconset resource="../icons/orbiticons.qrc">
+               <iconset>
```
Comment (dpallotti) [inline src/OrbitQt/orbitsamplingreport.ui:106]: I guess this is not intended? I'm wondering if this has unexpected effects.

File: src/OrbitQt/orbitsamplingreport.ui [hunk 2]
```diff
@@ -120,13 +120,16 @@
...
```
Comment (dpallotti) [inline src/OrbitQt/orbitsamplingreport.ui:123]: Same here.

File: src/OrbitQt/orbitsamplingreport.ui [hunk 3]
```diff
@@ -120,13 +120,16 @@
...
                 <normaloff>:/actions/keyboard_arrow_right</normaloff>:/actions/keyboard_arrow_right</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CallstackLabel">
+              <property name="toolTip">
```
Comment (dpallotti) [inline src/OrbitQt/orbitsamplingreport.ui:130]: I feel like this tooltip is very hidden as it's a tooltip for a label, I would not really expect that. But then again, all our tooltips are fairly hidden I guess. One option would be to add a dedicated label (e.g., below the existing one with a smaller font). Just some random thought.

File: src/OrbitGl/SamplingReport.cpp [hunk 4]
```diff
@@ -149,13 +150,36 @@ std::string SamplingReport::GetSelectedCallstackString() const {
   ORBIT_CHECK(callstack != nullptr);
   CallstackInfo::CallstackType callstack_type = callstack->type();
 
-  std::string type_string = (callstack_type == CallstackInfo::kComplete) ? "" : "  -  Unwind error";
+  std::string type_string =
+      (callstack_type == CallstackInfo::kComplete)
+          ? ""
+          : absl::StrFormat("  - Unwind error (%s)",
```
[the remark this answers is not in the document]
Comment (florian-kuebler) [reply src/OrbitGl/SamplingReport.cpp:156]: Oh, good catch.

File: src/OrbitQt/orbitsamplingreport.ui (hunk 1 above)
Comment (florian-kuebler) [reply src/OrbitQt/orbitsamplingreport.ui:106]: Arg. All changes in this file were done by QtDesigner... I'll revert them all.

**Pull request merged**